### DISCUSSION TITLE
I am unable to connect to the application without VPN using Cypress

### DISCUSSION THREAD

**Discussion by Prabhs2** [cypress-io/cypress#16336] [Questions and Help]
Hi, 

I am able to launch the application through Cypress behind VPN, but not without VPN. I have also tried setting proxy before running my Cypress scripts(without VPN) still it is not working and on launching the Cypress runner through Cypress open I am getting the below warning in the runner. 

Note: The application I am trying to access is (not locally available) on the server outside my system. 

>  Warning
> 
> Cypress could not verify that this server is running:
> 
> > https://<my_app_url(not localhost)>.com
> 
> This server has been configured as your baseUrl, and tests will likely fail if it is not running.

However, when the VPN is Off, I am manually able to launch the application on the browser, but the same is not happening through Cypress? 

Anybody facing a similar issue? or any feedback or suggestion will be of great help, this issue is blocking my CI integration.

**Comment by portyu9** [comment]
Hi @Prabhs2 

This is a proxy/network configuration issue, not `cy.visit()`.

Your normal browser is most likely reaching the application through a corporate proxy/PAC configuration that the Cypress process is not using.

Configure the proxy before starting Cypress:

```bash
export HTTPS_PROXY=http://your-company-proxy:port
export HTTP_PROXY=http://your-company-proxy:port

npx cypress open
```

Then verify what Cypress sees:

```bash
npx cypress info
```

If your company uses a PAC file, Cypress does not support PAC directly—you need the actual HTTP/HTTPS proxy address from your network team.

If the proxy performs TLS inspection, you may also need to configure the corporate CA certificate.

So the fix is to make the **Cypress process use the same network/proxy route as your normal browser**.

Please mark this answer as accepted if it helped.
